components/mpas-framework/Makefile

```diff
@@ -396,11 +396,11 @@ gnu-cray:
 	"FFLAGS_OPT = -O3 -m64 -ffree-line-length-none -fconvert=big-endian -ffree-form -ffpe-summary=none $${EXTRA_FFLAGS}" \
 	"CFLAGS_OPT = -O3 -m64" \
 	"CXXFLAGS_OPT = -O3 -m64" \
-	"LDFLAGS_OPT = -O3 -m64" \
+	"LDFLAGS_OPT = -O3 -m64 $(GNU_CRAY_LDFLAGS)" \
```

Comment:
@matthewhoffman, this environment variable (or argument to `make`) needs to be set to:
```
export GNU_CRAY_LDFLAGS="-Wl,--enable-new-dtags"
```
on Chicoma for now.  I'll make sure Compass and Polaris do this.  If someone is building for Chicoma outside of Compass or Polaris (good luck!), they would need to set this manually.

Are you okay with this fix?  I don't want to put in anything into the Makefile that tries to detect the machine or anything crazy like that.

Reply:
@xylar , this seems like the best solution given the circumstances